libs/common/src/types/azure-file-info.ts

```diff
@@ -0,0 +1,4 @@
+import { AzureFileInfoEntity } from '../sql-db-entities/azure-file-info';
+import { AuditableBaseEntity } from '../sql-db-entities/base-entities';
+
+export type AzureFileInfo = Omit<AzureFileInfoEntity, keyof AuditableBaseEntity | 'id' | 'utilisationReport'>;
```

Comment:
can we have a quick chat about this offline for me to check if this is what we might be trying going forwards etc.